### ISSUE TITLE
Manual @v1.5.1 10.5 Read and Write Files has a wrong import

### ISSUE THREAD

**Issue by MarvinGronhorst** [denoland/deno#8268]
There seems to be a wrong import in the manual: https://deno.land/manual@v1.5.1/examples/read_write_files#read , as far as i understand it has been there since std version 0.69.0
The manual describes the following: 
```javascript
import { readJsonSync } from "https://deno.land/std@0.76.0/fs/mod.ts";
import { fromFileUrl } from "https://deno.land/std@0.76.0/path/mod.ts";

function readJson(path: string): object {
  const file = fromFileUrl(new URL(path, import.meta.url));
  return readJsonSync(file) as object;
}
```

But there is no **readJsonSync** in https://deno.land/std@0.76.0/fs/mod.ts. Read and Write JSON have been removed from std/fs/mod.ts since https://deno.land/std@0.69.0/fs/mod.ts. 

Reading and writing JSON has been removed in the master version of the manual https://deno.land/manual@master/examples/read_write_files . 

Has **readJsonSync** been removed completely?

**Comment by MarvinGronhorst** [comment]
Oh they have been removed in #7255

**Comment by lucacasonato** [comment]
It seems deno.land/x/deno@v1.5.1 contains source code for v1.3.2: https://deno.land/x/deno@v1.5.1/std/version.ts

I must have messed up while publishing 1.5.1 source code. I will see if I can fix it. Otherwise it will be fixed in the 1.5.2 docs coming soon.

**Comment by lucacasonato** [comment]
Fixed in 1.5.2. https://deno.land/manual@v1.5.2/examples/read_write_files
